.github/workflows/health_checks.yml

```diff
@@ -164,6 +172,53 @@ jobs:
           aws-region: us-west-2
       - name: Run E2E tests
         run: npm run e2e
+  run_package_manager_tests_tests:
+    if: needs.do_include_package_manager_tests.outputs.run_package_manager_tests == 'true'
+    strategy:
+      # will finish running other test matrices even if one fails
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        pkg-manager: [npm, yarn-classic, yarn-modern, pnpm]
+        node-version: [20]
+        exclude:
+          - os: windows-latest
+            pkg-manager: pnpm
+    env:
+      PACKAGE_MANAGER: ${{ matrix.pkg-manager }}
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
```

Comment:
can timeout be smaller? how long these tests run ?

Reply:
changed to 25 min af8d396dfb